crates/tauri-utils/src/acl/schema.rs

```diff
@@ -93,6 +123,18 @@ pub trait PermissionSchemaGenerator<
   }
 }
 
+fn add_permissions_to_description(description: &str, permissions: &[String]) -> String {
```

Comment:
ideally this could be a default method on the trait `PermissionSchemaGenerator`

```rs
trait PermissionSchemaGenerator {
    fn default_set_description_with_permissions(&self) -> String;
        // then here you can use `self.default_set_description()`
        // and `self.default_set_permissions()`
    }
}
```

Reply:
This is also used for the permission sets, so I don't think it makes sense to have a function purely for the default description with permissions

Reply:
you already added the function, I am just saying it should be moved here and avoid passing arguments.

Reply:
Sorry I don't quite follow, this is used by both default permission set and normal permission sets, do you mean extract the default permission code into a function?

Reply:
Ah now I understand what you mean. we could make another trait but not worth it. this is fine.